<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-06-11T23:25:49 IST" name="My_Suite" finished-at="2023-06-11T23:26:02 IST" duration-ms="13021">
    <groups>
    </groups>
    <test started-at="2023-06-11T23:25:49 IST" name="ChromeBrowserTest" finished-at="2023-06-11T23:26:02 IST" duration-ms="13021">
      <class name="TestClass.TC_009_EndToEnd_Test">
        <test-method is-config="true" signature="Launch_Application()[pri:0, instance:TestClass.TC_009_EndToEnd_Test@60bd273d]" started-at="2023-06-11T23:25:49 IST" name="Launch_Application" finished-at="2023-06-11T23:25:57 IST" duration-ms="8183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Launch_Application -->
        <test-method signature="IndexPage_Test()[pri:1, instance:TestClass.TC_009_EndToEnd_Test@60bd273d]" started-at="2023-06-11T23:25:57 IST" name="IndexPage_Test" finished-at="2023-06-11T23:25:58 IST" duration-ms="1054" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IndexPage_Test -->
        <test-method signature="ProductSearchPage_Test()[pri:2, instance:TestClass.TC_009_EndToEnd_Test@60bd273d]" started-at="2023-06-11T23:25:58 IST" name="ProductSearchPage_Test" finished-at="2023-06-11T23:26:01 IST" duration-ms="2282" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Success: You have added HTC Touch HD to your shopping car!] but found [Success: You have added HTC Touch HD to your shopping cart!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Success: You have added HTC Touch HD to your shopping car!] but found [Success: You have added HTC Touch HD to your shopping cart!]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at TestClass.TC_009_EndToEnd_Test.ProductSearchPage_Test(TC_009_EndToEnd_Test.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductSearchPage_Test -->
        <test-method signature="ShopingCartPage_Test()[pri:3, instance:TestClass.TC_009_EndToEnd_Test@60bd273d]" started-at="2023-06-11T23:26:01 IST" name="ShopingCartPage_Test" finished-at="2023-06-11T23:26:01 IST" duration-ms="3" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "PageObjects.ShopingCartPage.GetUnitPrice(String)" because "this.ShopingCartPage" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "PageObjects.ShopingCartPage.GetUnitPrice(String)" because "this.ShopingCartPage" is null
at TestClass.TC_009_EndToEnd_Test.ShopingCartPage_Test(TC_009_EndToEnd_Test.java:73)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ShopingCartPage_Test -->
        <test-method is-config="true" signature="Application_Close()[pri:0, instance:TestClass.TC_009_EndToEnd_Test@60bd273d]" started-at="2023-06-11T23:26:01 IST" name="Application_Close" finished-at="2023-06-11T23:26:02 IST" duration-ms="690" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Application_Close -->
      </class> <!-- TestClass.TC_009_EndToEnd_Test -->
    </test> <!-- ChromeBrowserTest -->
  </suite> <!-- My_Suite -->
</testng-results>
